### PR TITLE
Can 614

### PR DESCRIPTION
**Pull request by shahab-ramzan**
Add camp statement UI added

### CODE REVIEW

File: src/components/ComponentPages/Statement/AddOrManage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        topic_num: 30,
+      };
+      const result = await getAllUsedNickNames(reqBody);
+      form.setFieldsValue({
+        nick_name: result?.data[0]?.id,
+      });
+      setNickNameData(result?.data);
+      setScreenLoading(false);
+    }
+    if (isLogin) {
```
Comment (akuks) [inline src/components/ComponentPages/Statement/AddOrManage/index.tsx]:
```javascript
(isLogin) ? nickNameListApiCall() : router.push("/login");
```

How about this?

File: src/components/ComponentPages/Statement/AddOrManage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                              {
+                                required: true,
+                                message: "Please select Namespace",
+                              },
+                            ]}
+                          >
+                            <Select defaultValue="/General/" size="large">
+                              <Select.Option value="general">
+                                /General/
+                              </Select.Option>
```
Comment (akuks) [inline src/components/ComponentPages/Statement/AddOrManage/index.tsx]: May be an array. Then iterate. I am pretty sure, we'll add more in this

File: src/components/ComponentPages/Statement/AddOrManage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+                      <Row gutter={24}>
+                        <Col xs={24} xl={12}>
+                          <Form.Item
+                            className={styles.formItem}
+                            name="statement"
+                            label={<>Statement </>}
+                            rules={[
+                              {
+                                required: true,
+                                message: "Statement is required",
```
Comment (akuks) [inline src/components/ComponentPages/Statement/AddOrManage/index.tsx]: . Must come from a common message file

File: src/components/ComponentPages/Statement/AddOrManage/index.tsx [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+                          label={
+                            <>
+                              Edit Summary{" "}
+                              <small>(Briefly describe your changes)</small>
+                            </>
+                          }
+                          rules={[
+                            {
+                              required: true,
+                              message: "Edit summary is required",
```
Comment (akuks) [inline src/components/ComponentPages/Statement/AddOrManage/index.tsx]: . Must come from a common message file

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/Statement/AddOrManage/index.tsx (hunk 1 above)
Comment (shahab-ramzan) [reply src/components/ComponentPages/Statement/AddOrManage/index.tsx]: Resolved in Next PR

**Pull request merged**